### ISSUE TITLE
Earmark txs on the Frontend (Onchain analytics)

### ISSUE THREAD

**Issue by Velenir** [gnosis/dex-react#517]
Add a distinguishing feature to transactions sent through the frontend

Can be a special ending to `gas` or `gasPrice`, e.g. 
```js
gas = gas + Number(web3c.utils.utf8ToHex('FE')).toString()
// gas = 1213.000 + '17989'
```

Easier to do for `gasPrice` as we already fetch it from a gas station

## Update

Add a tail to gas price

Small design of encoded data:
- browser
- screen size
- other?
- sentinel

### Estimation:
*3*

**Comment by anxolin** [comment]
A trivial implementation would be just to add this magic number.  So we can calculate the module on the gas/gasPrice and decide if this transaction was originated in the FE.

We could use a binary representation of a number, where every bit represents something:
i.e. Mobile: Yes/No

Although for modeling some other things, like user agent or wallet, it's more optimal to encode a number with a few bits. i.e. With 5 bit you can model 32 browsers.

**Comment by Velenir** [comment]
Closing as FE part is done and merged in #689
